monkestation/code/modules/mob/living/simple_animal/megafauna/stalwart.dm

```diff
@@ -0,0 +1,400 @@
+/**
+ * Stalwart, ported from yogstation
+ *
+ *
+ *
+ *
+ *
+ */
+
+/mob/living/simple_animal/hostile/megafauna/stalwart
```

Comment:
gonna let this slide since its a port, but for future notes simple mobs are sunset content and shouldn't have any more additions, the ai_controller system is way better